common/flavorization/00_CTP.txt

```diff
@@ -6526,6 +6526,400 @@ duke_tribal_female_italian = {
 ## End of Latin_cultrue_group ##
 ###############################################
 
+
+###############################################
+## Eastern_slavic_cultrue_group ##
+###############################################
+
+### Russian ###
+# Feudal
+emperor_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+emperor_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+empire_feudal_russian = {
+	type = title
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+title_prince_male_russian_empire = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = empire
+	priority = 40
+	cultures = { russian  }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+title_prince_female_russian_empire = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = empire
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
+}
+king_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+king_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+kingdom_russian = {
+	type = title
+	tier = empire
+	priority = 48
+	governments = { feudal_government tribal_government }
+	cultures = { russian }
+}
+title_prince_male_russian_kingdom = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = kingdom
+	priority = 40
+	cultures = { russian }
+	governments = { feudal_government tribal_government }
+	top_liege = no
```

Comment:
I think ```top_liege = yes``` (or just don't use it, its the default value) should be use for prince and princess.